cmd/troubleshoot/cli/run.go

```diff
@@ -183,6 +183,7 @@ func runTroubleshoot(v *viper.Viper, args []string) error {
 		OutputPath:                v.GetString("output"),
 		Redact:                    v.GetBool("redact"),
 		FromCLI:                   true,
+		RunHostCollectorsInPod:    false || mainBundle.Metadata.RunHostCollectorsInPod,
```

Comment:
Can this just be set to `mainBundle.Metadata.RunHostCollectorsInPod`?

Reply:
good catch, it has been set to `mainBundle.Metadata.RunHostCollectorsInPod`